viewing_party/party.py

```diff
@@ -1,23 +1,224 @@
 # ------------- WAVE 1 --------------------
 
+
+## Create a function that will create a movie function
+# with title, genre, and rating into a dictiionary
+# If there is no title, genre, or rating, return None
+# If any of the three variables is missing, return None
+
+from decimal import DivisionByZero
+from tests.test_constants import USER_DATA_2

```

Comment:
We generally don't want production code to import or otherwise be dependent on test data. It doesn't look like this import is being used now, so we would want to clean it up so our program doesn't bring in resources it doesn't need.